tardis/transport/montecarlo/packet_trackers.py

```diff
@@ -203,3 +204,23 @@ def track(self, r_packet):
         self.energy = r_packet.energy
         self.shell_id = r_packet.current_shell_id
         self.interaction_type = r_packet.last_interaction_type
+
+    # To make it compatible with RPacketTracker
+    def finalize_array(self):
+        pass
+
+
+@njit
+def rpacket_tracker_list(no_of_packets, length):
```

Comment:
this should be named generate_rpacket ...

Reply:
Sure.